### PR TITLE
Windowsd Server 2022: prepull docker images

### PR DESCRIPTION
**Pull request by viceice**
# Description
New tool, Bug fixing, or Improvement?  
Please include a summary of the change and which issue is fixed. Also include relevant motivation and context.  
**For new tools, please provide total size and installation time.**

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by damienpontifex** [comment]
@viceice want to also add them to virtual-environments/images/win/Windows2022-Readme.md similar to how they're noted on the readme for 2019 https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md

**Comment by viceice** [comment]
Ok, will update the readme

**Comment by viceice** [comment]
I'm not sure about the dates, just used today. Are this the image creation times and where do i get them easily?

File: images/win/Windows2022-Readme.md [hunk 1]
```diff
@@ -456,5 +456,14 @@ All other versions are saved but not installed.
...
 
+### Cached Docker images
+| Repository:Tag                                                            | Digest                                                                   | Created    |
+| ------------------------------------------------------------------------- | ------------------------------------------------------------------------ | ---------- |
+| mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2022  | sha256:38857a55ffcfc715fd4f79d1524db88ec0a8fefe11543db42a030424329b0bf6  | 2021-09-10 |
+| mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2022 | sha256:f449e95b12aa5e1c79681a5dd0dc35c4dcd95a2ca4c4568343b2b87cb6341084  | 2021-09-10 |
+| mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2022     | sha256:a3632ef8d3a55159f006cefd6306b832c67093564b1a89c1ae7e299299b7f175  | 2021-09-10 |
+| mcr.microsoft.com/windows/nanoserver:ltsc2022                             | sha256:3df3372895223b44475125ab3553fd39a71d89a896099d463d3216f1e22a3ab0  | 2021-09-10 |
+| mcr.microsoft.com/windows/servercore:ltsc2022                             | sha256:8f756a7fd4fe963cc7dd2c3ad1597327535da8e8f55a7d1932780934efa87e04  | 2021-09-10 |
```
Comment (viceice) [inline images/win/Windows2022-Readme.md:466]:
```suggestion
| mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2022  | sha256:38857a55ffcfc715fd4f79d1524db88ec0a8fefe11543db42a030424329b0bf6  | 2021-08-18 |
| mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2022 | sha256:f449e95b12aa5e1c79681a5dd0dc35c4dcd95a2ca4c4568343b2b87cb6341084  | 2021-08-18 |
| mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2022     | sha256:a3632ef8d3a55159f006cefd6306b832c67093564b1a89c1ae7e299299b7f175  | 2021-09-10 |
| mcr.microsoft.com/windows/nanoserver:ltsc2022                             | sha256:3df3372895223b44475125ab3553fd39a71d89a896099d463d3216f1e22a3ab0  | 2021-09-10 |
| mcr.microsoft.com/windows/servercore:ltsc2022                             | sha256:8f756a7fd4fe963cc7dd2c3ad1597327535da8e8f55a7d1932780934efa87e04  | 2021-09-10 |
```

Will update when i fetched others

**Comment by viceice** [comment]
Readme updated 🎉

**Comment by miketimofeev** [comment]
@viceice @damienpontifex there is no need to update the readme file as it's generated autmatically

**Review by al-cheb — DISMISSED** [review]

**Comment by viceice** [comment]
@miketimofeev Should i revert the readme chnages?

**Comment by miketimofeev** [comment]
@viceice yes, please

File: images/win/Windows2022-Readme.md [hunk 2]
```diff
@@ -456,5 +456,14 @@ All other versions are saved but not installed.
...
 
+### Cached Docker images
+| Repository:Tag                                                            | Digest                                                                   | Created    |
+| ------------------------------------------------------------------------- | ------------------------------------------------------------------------ | ---------- |
+| mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2022  | sha256:38857a55ffcfc715fd4f79d1524db88ec0a8fefe11543db42a030424329b0bf6  | 2021-08-18 |
+| mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2022 | sha256:f449e95b12aa5e1c79681a5dd0dc35c4dcd95a2ca4c4568343b2b87cb6341084  | 2021-08-18 |
+| mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2022     | sha256:a3632ef8d3a55159f006cefd6306b832c67093564b1a89c1ae7e299299b7f175  | 2021-08-18 |
+| mcr.microsoft.com/windows/nanoserver:ltsc2022                             | sha256:3df3372895223b44475125ab3553fd39a71d89a896099d463d3216f1e22a3ab0  | 2021-08-09 |
+| mcr.microsoft.com/windows/servercore:ltsc2022                             | sha256:8f756a7fd4fe963cc7dd2c3ad1597327535da8e8f55a7d1932780934efa87e04  | 2021-08-09 |
+
```
Comment (viceice) [inline images/win/Windows2022-Readme.md:467]:
```suggestion
```

**Comment by viceice** [comment]
done

**Review by al-cheb — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**